x-pack/test/security_solution_cypress/es_archives/exceptions/mappings.json

```diff
@@ -36,6 +36,10 @@
         "user": {
           "type": "nested",
           "properties": {
+            "name": {
```

Comment:
Can you explain what necessitated this mapping change? I couldn't find any reference to it here. Does this mapping exist in the real exceptions index, and if so, is it the only one we were missing from this archive?

Reply:
From other file
```    // this is a made-up index that has just the necessary
    // mappings to conduct tests, avoiding loading large
    // amounts of data like in auditbeat_exceptions
    ```
    
So I just, fix name, to be keyword as it used in my exception